mastracode/factory-ui/src/ui/domains/workspaces/components/sessionActivity.css

```
@@ -152,64 +152,194 @@
+  syntax: '<number>';
+  inherits: true;
+  initial-value: 0.5;
 }
 
 .session-wick {
   --belt-beat: 1.4s;
 
-  /* Lap length, in belt beats. Long, because the head is crossing a whole card
-   * rather than a 6px rail: it has to read as travelling, not as spinning. */
-  --wick-lap: 6;
+  --wick-phase: 0s;
```

Stylelint (17.14.0)

[error] Expected no empty line before custom property (custom-property-empty-line-before)

(custom-property-empty-line-before)